ci/check-documentation.py

```diff
@@ -79,6 +98,14 @@ def complain(message):
                     ' does not exist or is not a file')
                 continue
 
+            if md_path in md_dict:
+                readme_files.add(target_path)
+
+        if md_path in md_dict:
+            for link in md_dict[md_path]:
+                if link not in readme_files:
+                    complain(f"{link.relative_to(OMZ_ROOT)}: file is not referenced in {md_path.name}")
```

Comment:
The error is in the index file, not in the file that isn't referenced. So it's the index file that should go before the colon.